build/jest.transform.js

```diff
@@ -0,0 +1,4 @@
+var webpackConfig = require('./webpack.config.js')
+var babelConfig = webpackConfig.module.rules[0].use[0].query
```

Comment:
@dmitriiabramov You're right here, not so clean passing babel config directly from webpack that way. When extracting babelConfig to different file, do you think it outweighs `.babelrc` configuration file, or should `.babelrc` be used instead? Though, `.babelrc` configuration cannot be directly used in neither of these config files.